packages/core/config/adapters/defiLlama/oSnap.json

```diff
@@ -0,0 +1,38 @@
+{
+  "ethereum": {
+    "avatars": ["0xB524735356985D2f267FA010D681f061DfF03715", "0x90A48D5CF7343B08dA12E067680B4C6dbfE551Be"],
+    "tokens": [
```

Comment:
nit: any reason not to pre-emptively add WETH? Just since it's a super common token. Would also be nice to grab the ETH balances, but that would be a question for the adapter code, and probably wouldn't need to be included here.